recipes/pybamm/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "pybamm" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "c778f23c3354594dd509423d50570d154bc16daa6e63c2631ec6492f3f7f6b31"
+
+build:
+  number: 0
+  entry_points:
+    - pybamm_edit_parameter = pybamm.parameters_cli:edit_parameter
+    - pybamm_add_parameter = pybamm.parameters_cli:add_parameter
+    - pybamm_list_parameters = pybamm.parameters_cli:list_parameters
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True  # [py<36]
```

Comment:
This can be noarch if you make the python deps below `python >=3.6`

Reply:
pybamm depends on scikit-fem that requires python >= 3.6

Reply:
That's fine, just put `python >=3.6` below

Reply:
This should be noarch still

Reply:
I had to add a `skip` entry to skip to build on Windows, since no conda package is available for casadi on this platform.